blocks/init/src/Blocks/components/image/components/image-editor.js

```diff
@@ -56,7 +57,12 @@ export const ImageEditor = (attributes) => {
 								{(!imageUsePlaceholder) ?
 									<MediaPlaceholder
 										icon="format-image"
-										onSelect={(value) => setAttributes({ [`${componentName}Url`]: value.url })}
+										onSelect={(value) => {
+											setAttributes({
+												[`${componentName}Url`]: value.url,
+												[`${componentName}Alt`]: value.alt
```

Comment:
we should not store alt tag in the DB when saving the image. Because if you change alt in the media this will not change in any of the pages.

In the admin alt tag is not important only on frontend. If you need to fetch alt you can use this function https://developer.wordpress.org/reference/functions/attachment_url_to_postid/ to fetch post ID and attachment meta on the frontend.

Reply:
imageALT is ok to provide as a pro to a component but not to store it in the db

Reply:
The alt tag is stored in the database when you edit the image in the media library, as a post meta 👀 . So if you add it there, it will be pulled from the image automatically. And it's necessary to have it so that the screen readers can interpret the image (a11y).

And @JerrySpecter noticed that the `attachment_url_to_postid` function is not working all the time for some reason 🤷🏼‍♂️

Reply:
The way you did it is stored in the post_content field in the DB no the post_meta. This way on the frontend you will never get the updated alt if changed in the media